src/channel.ts

```diff
@@ -395,30 +395,21 @@ export class Channel<
    *
    * @param {string} messageID the message id
    * @param {Reaction<ReactionType, UserType>} reaction the reaction object for instance {type: 'love'}
-   * @param {string} [user_id] the id of the user (used only for server side request) default null
-   * @param {boolean} [enforce_unique] set true to overwrite existing reactions if any
+   * @param {{ enforce_unique?: boolean }} [options] Option object, {enforce_unique: true} to override any existing reaction
    *
    * @return {Promise<ReactionAPIResponse<AttachmentType, ChannelType, CommandType, MessageType, ReactionType, UserType>>} The Server Response
    */
   async sendReaction(
     messageID: string,
     reaction: Reaction<ReactionType, UserType>,
-    user_id?: string,
-    enforce_unique?: boolean,
+    options?: { enforce_unique?: boolean },
```

Comment:
missing user_id

Reply:
user_id is moved to reaction object

Reply:
I did intentionally similar to `sendMessage`.